python/gilded_rose.py

```diff
@@ -6,34 +6,39 @@ def __init__(self, items):
         self.items = items
 
     def update_quality(self):
+
         for item in self.items:
-            if item.name != "Aged Brie" and item.name != "Backstage passes to a TAFKAL80ETC concert":
-                if item.quality > 0:
-                    if item.name != "Sulfuras, Hand of Ragnaros":
-                        item.quality = item.quality - 1
-            else:
-                if item.quality < 50:
-                    item.quality = item.quality + 1
-                    if item.name == "Backstage passes to a TAFKAL80ETC concert":
-                        if item.sell_in < 11:
-                            if item.quality < 50:
-                                item.quality = item.quality + 1
-                        if item.sell_in < 6:
-                            if item.quality < 50:
-                                item.quality = item.quality + 1
             if item.name != "Sulfuras, Hand of Ragnaros":
                 item.sell_in = item.sell_in - 1
-            if item.sell_in < 0:
-                if item.name != "Aged Brie":
-                    if item.name != "Backstage passes to a TAFKAL80ETC concert":
-                        if item.quality > 0:
-                            if item.name != "Sulfuras, Hand of Ragnaros":
-                                item.quality = item.quality - 1
+
+            if item.name == "Aged Brie":
+                if item.quality < 50:
+                    item.quality = item.quality + 1
+
+            elif item.name == "Sulfuras, Hand of Ragnaros":
+                item.quality = item.quality
+
+            elif item.name.split(" ")[0] == "Backstage" and item.name.split(" ")[1] == "passes":
+                if item.sell_in > 10 and item.quality < 50:
+                    item.quality = item.quality + 1
+                elif 10 >= item.sell_in > 5:
+                    item.quality = item.quality + 2
+                elif 5 >= item.sell_in > 0:
+                    item.quality = item.quality + 3
+                else:
+                    item.quality = 0
+
+            elif item.name.split(" ")[0] == "Conjured":
+                item.quality = item.quality - 2
```

Comment:
It says that:

> "Conjured" items degrade in Quality twice as fast as normal items

That would mean the double amount, not by 2. Isn't there a case when that would not be 2? For example a "Conjured" item past its sell-in date.